djangocms_video/templates/djangocms_video/default/hls_stream_source.html

```diff
@@ -0,0 +1,64 @@
+{% load i18n cms_tags sekizai_tags %}
+
+{% if not disabled %}
+    {% with instance.hls_source_url as url %}
+        <source id="{{ source_id }}" src="{{ url }}" type="application/x-mpegURL" {{ instance.attributes_str }}>
+    {% endwith %}
+{% endif %}
+
+{% addtoblock "js" %}
+<script src="https://cdn.jsdelivr.net/npm/hls.js@1.5.17/dist/hls.min.js "></script>
```

Comment:
Could that source be configurable and default to the cdn? Alternatively, please add a comment to the README on how to overwrite this template.